src/containers/UsernameTranslation.js

```diff
@@ -0,0 +1,146 @@
+import React, { useState, useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link } from "react-router-dom";
+
+/*
+At the moment works ONLY with apiPath='seacat_auth'
+- userId (required, array) - user ids to be translated
+- apiPath - default (and only working so far) 'seacat_auth'
+- cleanupTime - default set to 24 hours
+
+Usage:
+
+import { Credentials } from 'asab-webui';
+...
+<Credentials app={props.app} id={['user1-id', 'user2-id', 'user3-id']} apiPath='seacat_auth' cleanupTime={1000 * 60 * 60 * 24 * 14}/>
+
+*/
+
+export function Credentials(props) {
+
+	const { t } = useTranslation();
+
+	const apiPath = props.apiPath ?? 'seacat_auth';
+	let API = props.app.axiosCreate(apiPath);
+
+	const userIdsArray = Array.isArray(props.userId) ? props.userId : [props.userId] ;
+
+	const cleanupTime = props.cleanupTime ?? 1000 * 60 * 60 * 24; // 24 hrs
+
+	const [usernames, setUsernames] = useState([]);
+
+	// asks the server for usernames, saves them to local storage and sets usernames to render
+	const retrieveUserNames = async () => {
+		try {
+			let response = await API.put(`usernames`, userIdsArray);
+			if (response.data.result !== "OK") {
+				throw new Error(t("ASABCredentials|Something went wrong, failed to fetch assigned credentials"));
+			}
+			const usernamesToLS = setUsernamesToLS(response.data.data, userIdsArray, cleanupTime);
+			setUsernames(usernamesToLS);
+		} catch (e) {
+			props.app.addAlert("warning", t("ASABCredentials|Something went wrong, failed to fetch assigned credentials"));
+		}
+	}
+
+	// compares array of IDs with data in localstorage
+	const matchUserIds = (credentials_id) => {
+		let currentTime = Date.now();
+		const usernamesInLS = getUsernamesFromLS('Credentials', cleanupTime);
+		let usernamesToRender = [];
+
+		if (usernamesInLS.credentials == undefined || usernamesInLS.credentials.length === 0 || usernamesInLS.expiration <= currentTime) {
```

Comment:
@petrKavulok You can use `Date.now()` directly here, instead of passing it in the variable, which is not used anywhere else (`currentTime`)

```
if (usernamesInLS.credentials == undefined || usernamesInLS.credentials.length === 0 || usernamesInLS.expiration <= Date.now())
```